pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc

```diff
@@ -1,19 +1,19 @@
 [id='pantheon-yaml-file_{context}']
 
-= The Pantheon configuration file
+= The {ProductShortName} configuration file
 
-You need to create a configuration file, named `pantheon2.yml`, in the root directory of your repository, to enable a Git repository as a source that Pantheon can use to import content. This yaml file provides necessary information that Pantheon uses to interact with files in the repository.
+You need to create a configuration file, named `pantheon2.yml`, in the root directory of your repository, to enable a Git repository as a source that {ProductShortName} can use to import content. This yaml file provides necessary information that {ProductShortName} uses to interact with files in the repository.
```

Comment:
Easy way to create a starter config file, assuming uploader is installed locally: `pantheon -S > pantheon2.yml`

Reply:
@benradey As you mentioned, you need to install the uploader tool locally to be able to use this command, so I didn't document it that way.